navigator_engine/common/graph_loader.py

```diff
@@ -132,6 +134,19 @@ def import_data(sheet_name, graphs):
                 model.db.session.add(action)
                 model.db.session.commit()
 
+                # Parse action's resource urls and add them to database
+                resources = _parse_resources(graph_data.at[idx, DATA_COLUMNS['ACTION_RESOURCES']])
+                for resource_row in resources:
+                    resource = model.Resource(
+                        title=resource_row['title'],
+                        url=resource_row['url'],
+                        action_id=action.id,

```

Comment:
It's not that important.... but just FYI You don't need to specify action ID, you can just attach the action and it will infer the action ID from it on commit.  The benefit of this is that you don't have to commit the action prior to creating the resources.

Reply:
I'll remove assigning `action_id` to keep the ORM usage consistent. The whole graph loader module would benefit from using this same ORM practice but that can be raised as a separate issue.